features/vpnregionselection/src/main/java/com/kape/vpnregionselection/ui/vm/VpnRegionSelectionViewModel.kt

```diff
@@ -135,37 +144,82 @@ class VpnRegionSelectionViewModel(
         val all = mutableListOf<ServerItem>()
         items?.let {
             for (server in it) {
-                all.add(
-                    ServerItem(
-                        type = ItemType.Content(
-                            isFavorite = isVpnServerFavorite(
-                                ServerData(
-                                    server.name,
-                                    server.isDedicatedIp,
+                if (settingsPrefs.isShowGeoLocatedServersEnabled()) {
```

Comment:
what about:
```
if (settingsPrefs.isShowGeoLocatedServersEnabled().not && server.isGeo) {
    continue
}
```
to avoid the duplication below with the `else`

Reply:
of course! I'll update.

Reply:
addressed.